Windows/Debugger/Debugger_Disasm.cpp

```diff
@@ -349,14 +394,16 @@ void CDisasm::runToLine()
 
 BOOL CDisasm::DlgProc(UINT message, WPARAM wParam, LPARAM lParam)
 {
+
 	//if (!m_hDlg) return FALSE;
-	switch(message)
+	switch (message)
```

Comment:
All this indentation change add a lot of noise to the diff (you may want to keep the old style).

Reply:
why ARE definhitions for functions here highlighted in blue and showing weird-ass line coordinates?? This is something i don't quite get.